packages/cactus-common/src/main/typescript/key-converter.ts

```diff
@@ -0,0 +1,128 @@
+import KeyEncoder from "key-encoder";
+
+export enum KeyFormat {
+  Raw = "raw",
+  Hex = "hex",
```

Comment:
I think original code https://github.com/blockstack/key-encoder-js defined this as 'der'. Can you use same value to avoid  confusion?

Reply:
Currently Cactus’s key converter does not support 'DER' format.  

Here's list of format and enum for the Cactus and the external key encoder representation.

PEM Format - Cactus' enum 'PEM', - Key-Encoder's enum 'PEM'
DER Format - Cactus not support, - Key-Encoder's enum 'DER'
HEX Format - Cactus' enum 'HEX', - Key-Encoder's enum 'RAW'
Buffer/Unit8Array Format - Cactus' enum 'RAW', - Key-Encoder not support.

Reply:
My comment intended to use a meaningful name for the format.
Actually key-encoder obviously supports DER(Distinguished Encoding Rule), which is official name for the format.
Please refer related description at https://github.com/blockstack/key-encoder-js#encoding-public-keys

Reply:
OK. Sorry for my misunderstanding.r
Is there any reason to eliminate the DER format which widely is used at may key management implementations.
If you are not willing to support it, I can approve this PR as is.